tests/index.spec.js

```diff
@@ -78,3 +78,54 @@ it("should replace el by aze", () => {
   });
   expect(code).toMatchSnapshot();
 });
+
+
+var exampleWithTemplates = `
```

Comment:
use `const` please 🙏 

```suggestion
const exampleWithTemplates = `
```

Reply:
didn't checked the date of the PR...

Sorry 😅 

I'll do it?

Thx for contribution